main.py

```diff
@@ -0,0 +1,100 @@
+import ee
+import argparse
+import geemap
+from geopy.geocoders import Nominatim
+from geopy.exc import GeopyError
+from decouple import config
+
+HEAT_MAP_GEE_PROJECT = config("GEE_PROJECT")
+ee.Initialize(project=HEAT_MAP_GEE_PROJECT)
+
+def get_city_coordinates()->list:
+    """Asks user for city"""
+    try: 
+        city_name = str(input("Enter the city urban center you want to view: "))
+        geolocator = Nominatim(user_agent="urban_heat_island")
+
+        location = geolocator.geocode(city_name)
+        if location:
+            city_coordinates = [location.latitude, location.longitude]
+            print(f"The center coordinates for your city are {city_coordinates}")
+            return city_coordinates
+        else:
+            raise ValueError(f"Location for '{city_name}' not found.")
+    
+    except GeopyError as e:
+        raise ValueError(f"Geolocation service error: {e}")
+
+def main(coordinates):
+    print("Hello from urban-heat-island!")
+    ## Approximate bounding box for Austin, Texas, creating a layer with legal boundraies 
+    city = ee.Geometry.Point(coordinates)
+    table = ee.FeatureCollection("FAO/GAUL/2015/level2")
+    roi = table.filterBounds(city).map(lambda vec: vec.simplify(1500)) # Region of Interest
```

Comment:
what is 1500 here? consider using constants for magic values:

Python Refactoring: Turn Magic Numbers into Constants for Readability
https://www.youtube.com/watch?v=IljQqLnPC14